packages/x-components/src/x-modules/search-box/components/search-input.vue

```diff
@@ -204,6 +211,20 @@
       this.$x.emit('UserPressedArrowKey', event.key as ArrowKey, this.createEventMetadata());
     }
 
+    /**
+     * Prevents entering blacklisted characters into the search input.
+     *
+     * @param event - The keyboard event with the key pressed.
+     *
+     * @internal
+     */
+    protected filterBlacklistedCharacters(event: InputEvent): void {
```

Comment:
We are not filtering the values, instead, we are "blocking" them. So, i will name it like:

`blockBlacklistedCharacters`, `avodiBlacklistedCharacters`, `preventBlacklistedCharacters`

what do you think?